src/main/main/java/Duke.java

```diff
@@ -46,16 +46,38 @@ public static void main(String[] args) {
                 list[taskNum-1].markAsDone();
                 System.out.println("____________________________________________________________\n"
                         + "Nice! I've marked this task as done:");
-                System.out.println(" [" + list[taskNum-1].getStatusIcon() + "] " + list[taskNum-1].description);
+                System.out.println(" " + list[taskNum-1].toString());
                 System.out.println("____________________________________________________________\n");
                 continue;
             }
+
+            if (line.toLowerCase().contains("todo")) {
+                String description = line.substring(5);
+                Todo t = new Todo(description);
+                list[listCount] = t;
+                listCount++;
+            } else if (line.toLowerCase().contains("deadline")) {
+                int deadlineDividerPosition = line.indexOf("/by");
+                String description = line.substring(9,deadlineDividerPosition);
+                String by = line.substring(deadlineDividerPosition+4);
+                Deadline t = new Deadline(description,by);
+                list[listCount] = t;
+                listCount++;
+            }   else if (line.toLowerCase().contains("event")) {
+                int eventDividerPosition = line.indexOf("/at");
+                String description = line.substring(6,eventDividerPosition);
+                String at = line.substring(eventDividerPosition+4);
+                Event t = new Event(description,at);
+                list[listCount] = t;
+                listCount++;
+            }
+
             System.out.println("____________________________________________________________\n"
```

Comment:
i think you can make a constant to take place of println("                          "\n), and this skips two lines?